lib/MapManager.js

```diff
@@ -0,0 +1,75 @@
+const fs = require("fs");
+const { exec } = require("child_process");
+
+/***
+ * stores current map to a backup folder
+ * @param name
+ */
+function storeMap(name, cb) {
+    if (!fs.existsSync("/mnt/data/valetudo/maps")) {
+        fs.mkdirSync("/mnt/data/valetudo/maps");
+    }
+
+    if (!fs.existsSync("/mnt/data/valetudo/maps/" + name)) {
+        fs.mkdirSync("/mnt/data/valetudo/maps/" + name);
+    }
+    if (!fs.existsSync("/mnt/data/rockrobo/last_map")) {
+        return cb("map doesn't exist");
+    }
+    fs.copyFileSync("/mnt/data/rockrobo/last_map", "/mnt/data/valetudo/maps/" + name + "/last_map");
+    if (fs.existsSync("/mnt/data/rockrobo/ChargerPos.data")) fs.copyFileSync("/mnt/data/rockrobo/ChargerPos.data", "/mnt/data/valetudo/maps/" + name + "/ChargerPos.data");
+    else {
+        // we don't have charger info
+    }
+    if (fs.existsSync("/mnt/data/rockrobo/PersistData_1.data")) fs.copyFileSync("/mnt/data/rockrobo/PersistData_1.data", "/mnt/data/valetudo/maps/" + name + "/PersistData_1.data");
+    if (fs.existsSync("/mnt/data/rockrobo/PersistData_2.data")) fs.copyFileSync("/mnt/data/rockrobo/PersistData_1.data", "/mnt/data/valetudo/maps/" + name + "/PersistData_2.data");
```

Comment:
PersistData_1 is copied twice.
Is this correct?

```suggestion
    if (fs.existsSync("/mnt/data/rockrobo/PersistData_2.data")) fs.copyFileSync("/mnt/data/rockrobo/PersistData_2.data", "/mnt/data/valetudo/maps/" + name + "/PersistData_2.data");
```